bridge/cypress/integration/uniform-integrations.spec.ts

```diff
@@ -169,6 +169,59 @@ describe('Integrations', () => {
     cy.location('pathname').should('eq', '/project/sockshop/uniform/services/355311a7bec3f35bf3abc2484ab09bcba8e2b297');
   });
 
+  it('should show an error message if can not parse shipyard.yaml', () => {
+    // given
+    cy.intercept('/api/project/sockshop/tasks', {
+      statusCode: 500,
+      body: 'Could not parse shipyard.yaml',
+    }).as('tasksResult');
+    cy.byTestId(uniformPage.UNIFORM_INTEGRATION_TABLE_LOC).find('dt-row').eq(1).click();
+    cy.get(uniformPage.SUBSCRIPTION_EXPANDABLE_LOC)
+      .first()
+      .find('dt-expandable-panel')
+      .byTestId('addSubscriptionButton')
+      .click();
+
+    cy.wait('@tasksResult');
+    cy.wait('@tasksResult');
+    cy.wait('@tasksResult');
```

Comment:
What is the reason for waiting 3 times?

Reply:
Because we still have the retry strategy for HTTP calls with status 500, and therefore the error message is shown after the result was not successful after 3 tries.